library/common-library/templates/_priority-class-name.tpl

```diff
@@ -0,0 +1,8 @@
+{{- /* Defines the pod priorityClassName */ -}}
+{{- define "common.priorityClassName" -}}
+    {{- if .Values.priorityClassName -}}
+        {{- .Values.priorityClassName -}}
+    {{- else if .Values.global.priorityClassName -}}
```

Comment:
Test here if global exists.

As we are in a library the values of the chart that are using it are unreliable. In fact, this is the thing why the idea of Guillermo on having a unit-test chart liked me so much because we could have an "empty values file" to test these kinds of things.